assets/loadtest/Makefile

```diff
@@ -76,3 +76,32 @@ forward-grafana:
 scale-auth:
 	kubectl scale deployment/teleport-auth --replicas=$(REPLICAS) -n teleport
 
+POSTGRES_NAME ?= rdspostgres
+POSTGRES_URL ?=
+MYSQL_NAME ?= rdsmysql
+MYSQL_URL ?=
+DATABASE_ROLE_ARN ?=
+TELEPORT_VERSION ?= 13
```

Comment:
Should we make this var more explicit like required as user input to prevent situation where the default 13 image version will be used ?